clipboard/src/definitions.ts

```diff
@@ -0,0 +1,43 @@
+declare module '@capacitor/core' {
+  interface PluginRegistry {
+    Clipboard: ClipboardPlugin;
+  }
+}
+
+export interface ClipboardPlugin {
+  /**
+   * Write a value to the clipboard (the "copy" action)
+   *
+   * @since 1.0.0
+   */
+  write(options: ClipboardWrite): Promise<void>;
+
+  /**
+   * Read a value from the clipboard (the "paste" action)
+   *
+   * @since 1.0.0
+   */
+  read(): Promise<ClipboardReadResult>;
+}
+
+/**
+ * Represents the data to be written to the clipboard.
+ *
+ * @since 1.0.0
+ */
+export interface ClipboardWrite {
+  string?: string;
+  image?: string;
+  url?: string;
+  label?: string; // Android only
```

Comment:
These should each have JSDoc blocks, as well 😬 

JS documentation tools ignore `//` comments. If we want users to know something like that, they should be in JSDoc comment blocks. Then it will come through in VS Code and be generated on our website.